modules/claims_api/spec/lib/claims_api/bd_spec.rb

```diff
@@ -133,8 +133,8 @@
           expect(json_body['data']['fileName']).to end_with('21-22a.pdf')
         end
 
-        it 'the claimId is nil' do
-          expect(json_body['data']['claimId']).to be_nil
+        it 'the claimId is not present' do
+          expect(json_body['data']).not_to have_key('claimId')
```

Comment:
do we need this duplication?

Reply:
I think so. One is for form 21-22; the other is for 21-22a.